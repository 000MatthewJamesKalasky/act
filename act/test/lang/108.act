defproc test()
{
   bool xr, xa;

   hse {
     a : xr- : b
||  
     p : xa+ : w 
   }
}

test t;
