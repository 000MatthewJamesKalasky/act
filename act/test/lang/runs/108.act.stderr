In expanding test (108.act:1)
In expanding ::<Global>
Error on or near line number 5.
FATAL: Missing target fragment labelled `b'!
